### DISCUSSION TITLE
Too many accounts

### DISCUSSION THREAD

**Discussion by USLTD** [directvt/vtm#224] [Q&A] [answered]
@o-sdn-o, I don't understand something.
Who are @monotty and @desktopio ? I get @netxs-group which is **„organization“** owning **vtm** but what about those 2? Are they @netxs-group's or your alt accounts?
I am highly confused 😓😓

**Comment by o-sdn-o** [comment, marked as answer]
These are all my accounts for different projects

  - `netxs-group` is something like an "organization" that unites all projects.
  - `monotty` is an alternate name for vtm and is also a Devanagari font (it is under development and awaits the GUI part of the vtm terminal). The vtm itself could be called monotty, but naming "vtm" is more appropriate, and it seems that this account will only be associated with the font.
  - `desktopio` is the name for the Desktopio Framework project that vtm is based on. This framework is not finished yet, so this account is hanging around idle for now.
  
It was also useful for me to try playing with these accounts in interaction with each other, for a better understanding of the github mechanics.
